earthfile2llb/listener.go

```diff
@@ -281,6 +281,8 @@ func (l *listener) ExitRunStmt(c *parser.RunStmtContext) {
 	withSSH := fs.Bool("ssh", false, "Make available the SSH agent of the host")
 	secrets := new(StringSliceFlag)
 	fs.Var(secrets, "secret", "Make available a secret")
+	secretFiles := new(StringSliceFlag)
+	fs.Var(secretFiles, "secret-file", "mount a secret as a file")
```

Comment:
Can you also expand build args in a for loop that looks like this one?

```
for i, s := range secrets.Args {
  secrets.Args[i] = l.expandArgs(s, true)
}
```

Reply:
good catch!